css/style.css

```diff
@@ -204,155 +171,46 @@ body {
   opacity: 1;
```

Comment:
From malden/css/style.css, the follow rule is missing:

```css
img.DialogBox-image {
  max-width: 100%;
}
```

Reply:
added at end of style.css